forge/db/views/DeviceGroup.js

```diff
@@ -0,0 +1,63 @@
+module.exports = function (app) {
+    app.addSchema({
+        $id: 'DeviceGroupSummary',
+        type: 'object',
+        properties: {
+            id: { type: 'string' },
+            name: { type: 'string' },
+            description: { type: 'string' },
+            model: { type: 'string' }
+        }
+    })
+    function deviceGroupSummary (group) {
+        // if (Object.hasOwn(group, 'get')) {
+        //     group = group.get({ plain: true })
+        // }
+        if (group.toJSON) {
+            group = group.toJSON()
+        }
+        const result = {
+            id: group.hashid,
+            name: group.name,
+            description: group.description,
+            deviceCount: group.deviceCount || 0
+        }
+        return result
+    }
+
+    app.addSchema({
+        $id: 'DeviceGroup',
+        type: 'object',
+        allOf: [{ $ref: 'DeviceGroupSummary' }],
+        properties: {
+            createdAt: { type: 'string' },
+            updatedAt: { type: 'string' },
+            DeviceGroup: { type: 'object', additionalProperties: true }
+        },
```

Comment:
This doesn't look quite right to me. This reads like the `DeviceGroup` object looks like this:

```
{
   id,
   name,
   description,
   model,
   createdAt,
   updatedAt,
   DeviceGroup: {
      ... some unknown set of properties
   }
}
```

That doesn't match what I see in the deviceGroup view function below which has `Application`, `deviceCount` and `devices` present at the top level and no `DeviceGroup` property.